docs/release-notes/8437-auth-refactor.rst

```diff
@@ -0,0 +1,17 @@
+:orphan:
+
+**Removed Features**
+
+-  **Breaking Change** We removed the accidentally-exposed Session object from the
+   ``determined.experimental.client`` namespace. It was never meant to be a public API and it was
+   not documented in :ref:`python-sdk`, but it was exposed in that namespace nonetheless. It was
+   also available as a deprecated legacy alias, ``determined.experimental.Session``. It is expected
+   that most users use the Python SDK normally and are unaffected by this change.
```

Comment:
It looks like you did not do this. If that was intentional, works for me. But if a mistake, I'm unresolving for your explicit "no thanks."

Reply:
I thought I did.  Will double-check, thank you.

Reply:
Yes, I did deal with this change.  The currently commited code says:

> It is expected that most users use the Python SDK normally and are unaffected by this change, **since the `det.experimental.client`'s `login()` and `Determined()` are unaffected.**

Where the bold part is my adaptation of your requested change.

(I have a special dislike for resolving comments without responding to them, so I'm glad you checked and I would have been very apologetic if that had been the case)